### PR TITLE
🔧 Bucket secret default

### PR DESCRIPTION
**Pull request by dankolbman**
Won't try to load the bucket secret from vault if not in environment, or the secret in vault is empty.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -90,7 +90,9 @@ def init_app(app):
         app.config['INDEXD_USER'] = indexd_secrets['data']['user']
         app.config['INDEXD_PASS'] = indexd_secrets['data']['password']
 
-        app.config['BUCKET_SERVICE_TOKEN'] = bucket_secrets['data']['token']
+        if bucket_secrets:
+            app.config['BUCKET_SERVICE_TOKEN'] = \
+                    bucket_secrets['data']['token']
```
Comment (znatty22) [inline config.py]: Will data and token keys always exist?

Comment (dankolbman) [reply config.py]: They should as long as bucket secrets comes back non-null

Comment (dankolbman) [reply config.py]: But I will make it more defensive

**Review by alubneuski — APPROVED** [review]

**Review by znatty22 — APPROVED** [review]
👍

**Pull request merged**